public/static/partials/github_commit_panel.html

```diff
@@ -10,7 +10,7 @@
         ([[commit.author_email]])
       </span>
       <span class="muted">
-        authored at [[commit.ingest_time | convertDateToUserTimezone:timezone:"MMM D, YYYY h:mm:ss a"]]
+        authored at [[(commit.ingest_time || commit.create_time) | convertDateToUserTimezone:timezone:"MMM D, YYYY h:mm:ss a"]]
```

Comment:
Since commit.ingest_time isn't meaningful, this should _only_+ be commit.create_time (or say something like "unknown" if that value isn't available).